apps/src/gamelab/dropletConfig.js

```diff
@@ -41,26 +41,26 @@ function chooseAsset(typeFilter, callback) {
 module.exports.blocks = [
   // Game Lab
   {func: 'draw', block: 'function draw() {}', expansion: 'function draw() {\n  __;\n}', category: 'Game Lab', noAutocomplete: true, customDocURL: "http://p5js.org/reference/#/p5/draw" },
-  {func: 'drawSprites', category: 'Game Lab', customDocURL: "http://p5play.molleindustria.org/docs/classes/p5.play.html#method-updateSprites" },
-  {func: 'allSprites', category: 'Game Lab', type: 'readonlyproperty', noAutocomplete: true, customDocURL: "http://p5play.molleindustria.org/docs/classes/p5.play.html#prop-allSprites" },
+  {func: 'drawSprites', category: 'Game Lab', customDocURL: "https://code-dot-org.github.io/p5.play/docs/classes/p5.play.html#method-drawSprites" },
+  {func: 'allSprites', category: 'Game Lab', type: 'readonlyproperty', noAutocomplete: true, customDocURL: "https://code-dot-org.github.io/p5.play/docs/classes/p5.play.html#prop-allSprites" },
   {func: 'Game.allSprites', category: 'Game Lab', type: 'readonlyproperty' },
   /* disabled since we aren't suggesting these global properties be used - commenting these out prevents droplet from turning 'width' and 'height' into blocks when referenced as locals or parameters */
-  //  {func: 'width', category: 'Game Lab', type: 'readonlyproperty', noAutocomplete: true, customDocURL: "http://p5js.org/reference/#/p5/width" },
-  //  {func: 'height', category: 'Game Lab', type: 'readonlyproperty', noAutocomplete: true, customDocURL: "http://p5js.org/reference/#/p5/height" },
-  {func: 'Game.width', category: 'Game Lab', type: 'readonlyproperty' },
-  {func: 'Game.height', category: 'Game Lab', type: 'readonlyproperty' },
-  {func: 'Game.mouseX', category: 'Game Lab', type: 'readonlyproperty' },
-  {func: 'Game.mouseY', category: 'Game Lab', type: 'readonlyproperty' },
-  {func: 'Game.frameRate', category: 'Game Lab', type: 'property' },
-  {func: 'Game.frameCount', category: 'Game Lab', type: 'readonlyproperty' },
+  //  {func: 'width', category: 'Game Lab', type: 'readonlyproperty', noAutocomplete: true },
+  //  {func: 'height', category: 'Game Lab', type: 'readonlyproperty', noAutocomplete: true },
+  {func: 'Game.width', category: 'Game Lab', type: 'readonlyproperty', customDocURL: "http://p5js.org/reference/#/p5/width" },
+  {func: 'Game.height', category: 'Game Lab', type: 'readonlyproperty', customDocURL: "http://p5js.org/reference/#/p5/height" },
+  {func: 'Game.mouseX', category: 'Game Lab', type: 'readonlyproperty', customDocURL: "http://p5js.org/reference/#/p5/mouseX" },
+  {func: 'Game.mouseY', category: 'Game Lab', type: 'readonlyproperty', customDocURL: "http://p5js.org/reference/#/p5/mouseY" },
+  {func: 'Game.frameRate', category: 'Game Lab', type: 'property', customDocURL: "http://p5js.org/reference/#/p5/frameRate" },
+  {func: 'Game.frameCount', category: 'Game Lab', type: 'readonlyproperty', customDocURL: "http://p5js.org/reference/#/p5/frameCount" },
```

Comment:
Here and in several other places I added documentation links where we had none.  Note that we're still fine with external links to p5js.org (which is kid-friendly and whose docs we haven't modified) - just not molleindustria.org.